tests/validators/tests.py

```diff
@@ -222,6 +222,9 @@
     (URLValidator(EXTENDED_SCHEMES), 'git+ssh://git@github.com/example/hg-git.git', None),
 
     (URLValidator(EXTENDED_SCHEMES), 'git://-invalid.com', ValidationError),
+
```

Comment:
Chop blank line.

Reply:
Applied all the changes as you said in the next commit. Please review and let me know if anything else also needs to be changed.